charts/rbac/templates/developers/mongodb.yaml

```diff
@@ -0,0 +1,31 @@
+{{ if .Values.rbac.developers.mongodb }}
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+ name: saritasa-developers-role
+ namespace: mongodb
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods/portforward", "pods/exec"]
+    verbs: ["create"]
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: saritasa:developers
+  namespace: mongodb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: saritasa-developers-role
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: saritasa:sso:developers
+
+{{ end }}
```

Comment:
I don't think they should have access into any DB namespace like mongodb or mysql or influx etc. Because these namespaces could contain pods running the actual DB, i.e. in rocks cluster:

```
➜ kgpon mongodb                                         
NAME                                     READY   STATUS    RESTARTS   AGE
mongodb-0                                2/2     Running   6          13d
saritasa-rocks-mongodb-5f669ff7d-hn96z   2/2     Running   0          89d
```

In production we probalbly need to adjust implementation of utilities in the namespace after the DB engine, i.e. instead of placing psql and psql port forward in "postgres" namespace - we should be placing them in the "utils" namespace.

Reply:
But they need to make DB dumps sometimes, so I think they should have these permissions

Reply:
that's what I say - they should be able to do that via CLI using these utils pods

```
➜ kgpon postgres
NAME                                              READY   STATUS    RESTARTS   AGE
tccfapi-prod-psql-75c7bc98bd-7jsht                2/2     Running   0          44d
tccfapi-prod-psql-port-forward-66bcf7c8dc-nf87b   1/1     Running   0          44d
```

but these pods should live in "utils" pods namespace. So we should introduce new ns as part of our boilerplate **utils** that will have these type of pods.

Instead of having them inside "postgres" (or mysql or mongodb, or influxdb or etcd etc pods)
that way they can't login inside mongodb server pod (to do dump) and damage the stability

So for example - I can login to mongodb server pod and run mongodb for the DB that is greater than the allocated storage === DOWNTIME.

If you're not clear - pls ping me your morning so we can continue the discussion further.

We can accept the current PR as is, but then a new issue has to be created where we move the "utility" type pods into utility NS and grant RBAC roles for **developers** group only to that utilities namespace.